templates/logstash.service.j2

```diff
@@ -0,0 +1,28 @@
+# {{ ansible_managed }}
```

Comment:
It's really bad practice to hardcode variables. Add them to defaults and put here